hopr/hopr-lib/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hopr-lib"
-version = "3.0.0"
+version = "3.1.0"
```

Comment:
This cannot iterate, since it correlates with the actual release, so release v3 must start at 3.0.0